PWGHF/DataModel/CandidateSkimmingTables.h

```diff
@@ -0,0 +1,323 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file CandidateSkimmingTables.h
+/// \brief Definitions of tables produced by the candidate skimming
+///
+/// \author Gian Michele Innocenti <gian.michele.innocenti@cern.ch>, CERN
+/// \author Vít Kučera <vit.kucera@cern.ch>, CERN
+
+#ifndef PWGHF_DATAMODEL_CANDIDATESKIMMINGTABLES_H_
```

Comment:
I would refrain from using the term "skimming" to avoid confusion, since in ALICE skimming is referring to the software triggers (I know that we called "skimCreator" our HF vertex finder, but this would also not be a proper name, since we are not skimming anything, but actually adding information to the AO2Ds). I propose to use the `TrackIndexSkimCreatorTables.h` or something similar, to make it a bit more clear that it refers to the `trackIndexSkimCreator` task.

Reply:
I think it's not "skimming" because if it was it, it would imply a reduction of the information while in our case we add the information of candidates starting from tracks (so it's vertex finding, similar to the SVertexer in the reconstruction). In any case I think that `TrackIndexSkimmingTables` makes it less ambiguous, thanks!